pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -54,8 +56,9 @@ func NewDeviceCheck(config *checkconfig.CheckConfig, ipAddress string, sessionFa
 	}
 
 	return &DeviceCheck{
-		config:  newConfig,
-		session: sess,
+		config:            newConfig,
+		session:           sess,
+		sessionCloseError: atomic.NewUint64(0),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
```suggestion
		sessionCloseErrorCount: atomic.NewUint64(0),
```

Reply:
Fixed here https://github.com/DataDog/datadog-agent/pull/14350/commits/90f98d9682497b222ff8777ed4fb6a2aca319721